### PR TITLE
Add new functionality

### PR DESCRIPTION
**Pull request by Apostoln**
- Get children devices (partitions) for disk
 - Get property value for device

Part of [#698](https://github.com/elastio/elastio/issues/698)

### CODE REVIEW

**Comment by Apostoln** [comment]
@anelson
Note: This is a PR to fake branch for your review.

I'll create a PR to upstream after your approve.

**Comment by Apostoln** [comment]
Note: I've missed that getting disk(parent device) for partition is already implemented in block-utils. So I've just implemented only  getting partitions(children) for disk and getting device's tags(properties).

File: src/lib.rs [hunk 1]
```diff
@@ -1204,6 +1205,56 @@ pub fn is_block_device(device_path: impl AsRef<Path>) -> BlockResult<bool> {
     )))
 }
 
+/// Get property value by key `tag` for device with devpath `device_path` (like "/dev/sda") if present
+#[cfg(target_os = "linux")]
+pub fn get_block_dev_property(
+    device_path: impl AsRef<Path>,
+    tag: &str,
+) -> BlockResult<Option<String>> {
+    let error_message = format!(
```
Comment (anelson) [inline src/lib.rs]: Why incur the mem allocate and overhead for this message before you even have an error?  You could do this `format!` in the closure that creates the `BlockUtilsError`

File: src/lib.rs [hunk 2]
```diff
@@ -1204,6 +1205,56 @@ pub fn is_block_device(device_path: impl AsRef<Path>) -> BlockResult<bool> {
...
+        .property_value(tag)
+        .map(|value| value.to_string_lossy().to_string()))
+}
+
+/// Get properties for device with devpath `device_path` (like "/dev/sda") if present
+#[cfg(target_os = "linux")]
+pub fn get_block_dev_properties(
+    device_path: impl AsRef<Path>,
+) -> BlockResult<HashMap<String, String>> {
+    let error_message = format!(
```
Comment (anelson) [inline src/lib.rs]: Same wasteful approach here

**Review by anelson — CHANGES_REQUESTED** [review]

**Review by anelson — APPROVED** [review]

**Pull request merged**